src/sender.js

```diff
@@ -131,7 +136,10 @@ async function main({ msg, ackMsg, nackMsg, sendToQueue, channel }) {
       }
     })
 
+    logger.debug('Updating nonce')
     await updateNonce(nonce)
+
+    logger.debug('Releasing lock')
     await lock.unlock()
 
     if (failedTx.length) {
```

Comment:
should it be `warn` or `error`?

Reply:
Releasing the lock is the normal flow of execution, I don't see why it should be a warning or an error.